### PR TITLE
Add the link to Fun expressions manual page

### PR DESCRIPTION
**Pull request by hirotnk**
The current document does not show full description of fun syntax.

I got help from @nox to see if the syntax with optional function name is legitimate, so hopefully this PR will help to make the document less confusing.

### CODE REVIEW

**Comment by nox** [comment]
Good initiative @hirotnk. May I suggest putting a space before the opening parenthesis and replacing the capital 'R' of 'read' by 'r'?

**Comment by hirotnk** [comment]
Thank you @nox for reviewing, this was done.

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

File: system/doc/programming_examples/funs.xmlsrc [hunk 1]
```diff
@@ -115,7 +115,7 @@ foreach(fun(Pid) -> Pid ! M end, L)</code>
 
   <section>
     <title>The Syntax of Funs</title>
-    <p>Funs are written with the syntax:</p>
+    <p>Funs are written with the syntax (read more for the full description in <seealso marker="doc/reference_manual:expressions#funs">Fun Expressions</seealso>):</p>
```
Comment (zhird) [inline system/doc/programming_examples/funs.xmlsrc]: Maybe "see <seealso.....</seealso> for full description):"? 

Also, please add line breaks to match the rest of the file.

**Comment by hirotnk** [comment]
@zhird I modified the commit as your request. Does it work ?

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

**Comment by zhird** [comment]
Yes that's better. I would remove the "the" in "for the full description" though.

**Comment by bjorng** [comment]
We have made extensive changes to the system documentation in the 'master' branch, so we want to avoid improvements to the system documentation in 'maint' branch.

Please rebase your branch on the 'master' branch.

**Comment by hirotnk** [comment]
@zhird Will do.
@bjorng I'd like to close this PR and make this change out of 'master' branch. Does that sound OK ?

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

**Comment by bjorng** [comment]
OK. I'll close this one. We will wait for your new pull request.

**Pull request closed without merging**